### PR TITLE
ENH add code to copy outputs

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Pushed the branch to main repo
* [ ] CI passed on the branch

This is an experimental block of code to start collecting outputs on a single repo and see how badly github fails.

### CODE REVIEW

File: conda_forge_webservices/webapp.py [hunk 1]
```diff
@@ -527,6 +527,25 @@ def _do_copy(feedstock, outputs, win_only, channel, git_sha):
             outputs_to_copy,
             channel,
         )
+        if "REGRO_GITHUB_TOKEN" in os.environ:
+            try:
+                gh = github.Github(os.environ["REGRO_GITHUB_TOKEN"])
+                repo = gh.get_repo("regro/releases")
```
Comment (isuruf) [inline conda_forge_webservices/webapp.py:533]: Why put this in `regro` and not `conda-forge`?

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: It is very experimental and I may blow away the repo later. I am not 100% sure how to do this yet.

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: I don't want to set any expectations that it will be around later.

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: Once we ship for real, it will def be in conda-forge.

Comment (isuruf) [reply conda_forge_webservices/webapp.py:533]: Can it be a bot where there's shared access?

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: What do you mean?

Comment (isuruf) [reply conda_forge_webservices/webapp.py:533]: `conda-forge-bot/releases` instead of `regro/releases`

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: I am using the conda-forge-daemon bot with a key in keybase. So core does have access.

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: Ahh I see. The `conda-forge-daemon` bot already has write access, so ppl can pull the key for it and access the repo. This is basically the same as what you are proposing.

Comment (isuruf) [reply conda_forge_webservices/webapp.py:533]: Even that bot does not have admin access to the `regro/releases` repo. Neither do I

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: I'm also happy to add anyone who'd like from core directly on the repo.

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: cc @conda-forge/core

Comment (beckermr) [reply conda_forge_webservices/webapp.py:533]: I've added both the bot and you @isuruf as admins.

**Comment by beckermr** [comment]
test failure is due to smithy bug: https://github.com/conda-forge/conda-forge-webservices/pull/16#issuecomment-674172694

**Pull request merged**